add Jupyter Notebook to overview (#837)
MicrosoftDocs/quantum-docs-pr: articles/overview/what-is-qsharp-and-qdk.md

- Running [Q# from the command line](xref:microsoft.quantum.install.standalone) is the quickest way to get started.

> - Run standalone Q# programs using [Jupyter Notebooks with IQ#](xref:microsoft.quantum.install.jupyter), an extension to the .NET Core SDK used by Jupyter and Python to provide the core functionality for compiling and simulating Q# operations.

Author: ```suggestion
- Run standalone [Jupyter Notebooks with IQ#](xref:microsoft.quantum.install.jupyter), a Jupyter extension for compiling, simulating, and visualizing Q# programs.
```
